app/Http/Controllers/NoteController.php

```diff
@@ -14,7 +14,7 @@ public function index()
     {
```

Comment:
When I run the application, it prompts me to log in. I create a new account then I'm always forwarded to `/verify-email`. I didn't get an e-mail. How do I get past this to use the application?